doc/nrf/device_guides/working_with_nrf/nrf53/features_nrf53.rst

```diff
@@ -0,0 +1,78 @@
+.. _nrf53_features:
+
+Features of nRF53 Series
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF53 Series System-on-Chips (SoC) integrate a high-performance Arm® Cortex®-M33 dual-core processor with a 2.4 GHz RF transceiver, in addition to advanced security features.
+All nRF53 Series SoC support Bluetooth® 5.1 and Bluetooth Mesh, in addition to multiprotocol capabilities.
+
+For additional information, refer to the following resources:
+
+* `nRF53 Series`_ for the technical documentation on the nRF53 Series chips and associated kits.
+* :ref:`ug_nrf5340_gs` for getting started with the nRF5340 DK.
+* :ref:`ug_nrf5340` for advanced topics specific to the nRF5340 DK.
+* :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
```

Comment:
```suggestion
* `nRF53 Series`_ for the technical documentation on the nRF53 Series chips and associated kits.
* :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.

For the nRF5340 DK, see the following user guides:

* :ref:`ug_nrf5340_gs` for getting started with the nRF5340 DK.
* :ref:`ug_nrf5340` for advanced topics specific to the nRF5340 DK.

```
Since the Thingy:53 has its own sections as well.